### PR TITLE
Fix wrong flags to osm install in OSM Manual Demo Guide

### PR DESCRIPTION
**Pull request by eoftedal**

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Hi, @eoftedal! We'll review this PR, but meanwhile please sign your commit so we can merge if accepted? https://github.com/openservicemesh/osm-docs/pull/58/checks?check_run_id=2618664431 for details. Thanks!

File: content/docs/install/manual_demo.md [hunk 1]
```diff
@@ -54,18 +54,16 @@ This command enables
 [Prometheus](https://github.com/prometheus/prometheus),
 [Grafana](https://github.com/grafana/grafana), and
 [Jaeger](https://github.com/jaegertracing/jaeger) integrations.
-The `OpenServiceMesh.enablePermissiveTrafficPolicy` chart value instructs OSM to ignore any policies and
```
Comment (shashankram) [inline content/docs/install/manual_demo.md:57]: This is correct, but referring to the chart parameter in the `values.yaml` file. I think we should clarify this.

File: content/docs/install/manual_demo.md [hunk 2]
```diff
@@ -54,18 +54,16 @@ This command enables
...
 > Note: Permissive Traffic Policy mode is an important feature for brownfield deployments, where it may take some time to craft SMI policies. While operators design the SMI policies, existing services will to continue to operate as they have been before OSM was installed.
 
 ```bash
 osm install \
-    --set=OpenServiceMesh.enablePermissiveTrafficPolicy=true
-    --set=OpenServiceMesh.deployPrometheus=true \
-    --set=OpenServiceMesh.deployGrafana=true \
-    --set=OpenServiceMesh.deployJaeger=true
+    --enable-permissive-traffic-policy \
+    --deploy-grafana  --deploy-jaeger --deploy-prometheus
```
Comment (shashankram) [inline content/docs/install/manual_demo.md:66]: This is no longer true. The previous revision is correct.

**Review by shashankram — CHANGES_REQUESTED** [review]
Hey @eoftedal these changes you made are applicable to the v0.8 release, but not meant for the `main` branch. In `main`, the commands use the `--set` option.

**Comment by bridgetkromhout** [comment]
Closing due to merge conflicts; superceded by https://github.com/openservicemesh/osm-docs/pull/103.

**Pull request closed without merging**